packages/mds-policy-service/service/validators.ts

```diff
@@ -104,74 +86,128 @@ const checkRateFieldsIfConditionSchema = () => ({
   },
   then: {
     properties: {
-      currency: stringSchema()
+      currency: { type: 'string' }
     },
     required: ['currency']
   }
-})
+}
+
+type MICRO_MOBILITY_VEHICLE_EVENT_ARRAY_ENUM = {
+  type: 'array'
+  items: { type: 'string'; enum: MICRO_MOBILITY_VEHICLE_EVENT[] | [] }
```

Comment:
Semicolon seems funny, surprised the linter isn't removing it (might just be because this is a type, though)